src/components/controls/toggle-focus.tsx

```diff
@@ -38,8 +38,15 @@ function ToggleFocus({ tooltip, focus, layout, dispatch, mobileDisplay }: {
   return (
     <Toggle
       display
-      on={focus}
-      callback={() => dispatch({ type: TOGGLE_FOCUS })}
+      on={focus === "selected"}
```

Comment:
There are two different approaches:

1. Keep `focus` as boolean internally, and `"selected"`/unset in the URL query: a6c0176d5080c49e201445443cc341f6eb8ad29c...55ebcc2e0fb99881d99ab6eceb534e7f2c4e35ef
2. Use `"selected"`/unset everywhere: 84fbc27ea45d70a452c90017d2dd03c41d5e029b...7e3a69f0c7362969533509611daa221206ae28ae

This PR was initially (1) but I've updated it to (2). I think it makes sense to do the upfront work to convert the internal representation if we are introducing a different external representation, even if it's not really necessary at the moment.